app-backup/luckybackup/luckybackup-0.4.9.recipe

```diff
@@ -0,0 +1,93 @@
+SUMMARY="A powerful, fast and reliable backup & sync tool"
+DESCRIPTION="An application that backs-up and/or synchronizes any directories with the power \
+of rsync. It is simple to use, fast (transfers over only changes made and not all data), safe\
+ (keeps your data safe by checking all declared directories before proceeding in any data \
+manipulation), reliable and fully customizable."
+COPYRIGHT="2008-2017 Loukas Avgeriou"
+LICENSE="GNU GPL v3"
+REVISION="1"
+HOMEPAGE="http://luckybackup.sourceforge.net"
+SOURCE_URI="https://downloads.sourceforge.net/luckybackup/luckybackup-$portVersion.tar.gz"
+CHECKSUM_SHA256="c161e98a11de5ad0a32a8e060e3fd45695a93ea3adf0581764a6b5d32e8da23b"
+ADDITIONAL_FILES="luckybackup.rdef.in"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+ SECONDARY_ARCHITECTURES="x86"
+fi
+
+PROVIDES="
+	luckybackup$secondaryArchSuffix = $portVersion
+	app:LuckyBackup = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
+	lib:libQt5Core$secondaryArchSuffix
+	lib:libQt5Gui$secondaryArchSuffix
+	lib:libQt5Network$secondaryArchSuffix
+	lib:libQt5WebKit$secondaryArchSuffix
+	lib:libQt5Widgets$secondaryArchSuffix
+	lib:librsync$secondaryArchSuffix
+	lib:libssl$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libGL$secondaryArchSuffix
+	devel:libQt5Core$secondaryArchSuffix
+	devel:libQt5Gui$secondaryArchSuffix
+	devel:libQt5Network$secondaryArchSuffix
+	devel:libQt5WebKit$secondaryArchSuffix
+	devel:libQt5Widgets$secondaryArchSuffix
+	devel:librsync$secondaryArchSuffix
+	devel:libssl$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:qmake$secondaryArchSuffix >= 5
+	"
+
+BUILD()
+{
+	qmake
```

Comment:
Thanks for the update!

I saw that there is a `luckybackup.pro` file. I guess it could be passed to `qmake`:
```diff
- 	qmake
+	qmake luckybackup.pro
```

BTW, last time I built the app I noticed that the License tab of the **_About_** box displays an error saying it cannot find the license. It would be nice to fix this :)

Maybe patching `luckybackup.pro` with `sed` in `PATCH` would allow to fix the "License not found" issue described above.
(If you need some hints for this I can guide you.)

Other possible improvements could be to move some files to more standard directories.
The documentation, for example, would be better in `"$docDir"` instead of having them in a subdirectory of `"$appsDir"/LuckyBackup/`.
OK, this is polishing, and we can do this later if you prefer.

Reply:
ok done.. 
for about box. . i can edit that. .but i afraid that .. if other platform show ok but not on haiku.. maybe its related to other bug.. qt issue maybe.. 
what do you think?

Reply:
sorry.. i misunderstand your meaning.. ok i'll check cannot found license issue..

Reply:
i've been update recipe.. it is ok when execute from terminal on path installed (/system/apps/LuckyBackup/LuckyBackup)
license show... but not when executed via menu haiku

Reply:
Oops, I'm still getting the error: `Could not locate the license file!!`
I believe this can be fixed by telling qmake where the license should be, using `make install` to install it and leave it there. Installing the license file with cp is probably what prevents the app from finding it because it is probably looking in `/usr/share/doc/luckybackup/license/`.
Maybe something like:
```bash
PATCH()
{
	sed -i \
		-e "/^license.path =/ s|=.*|= $docDir|;" \
		luckybackup.pro
}
```
could help, even if turns out that we cannot use `qmake install` to install it.
So maybe patching `luckybackup.pro` and modifying the target directory of the `cp` in `INSTALL` can do the trick.